src/content/apis/exchange-rate/exchangerate-api.md

```diff
@@ -0,0 +1,10 @@
+---
+name: ExchangeRate-API
+description: The Accurate & Reliable Exchange Rate API
+categories:
+  - exchange-rate
```

Comment:
I believe exchange-rate is too specific, I believe it's better to have a specific category like currency
```suggestion
  - currency
```

Reply:
do I have to change the name of the folder to currency also?

Reply:
Yes please

Reply:
Done